agent-control/src/cli/install.rs

```diff
@@ -38,6 +38,14 @@ pub struct InstallData {
     #[arg(long)]
     pub chart_version: String,
 
+    /// Name of the Helm release
+    ///
+    /// WARNING: Overriding the release name when installing Agent Control
+    /// will break the health checker and version updater. Both mechanisms
+    /// expect the release name to be `agent-control-deployment`.
+    #[arg(long)]
+    pub release_name: Option<String>,
```

Comment:
If we support configuring this, shouldn't the health-checker and version-checker receive such value through Agent Control configuration? Is this warning actually a TODO?